deps/build.jl

```diff
@@ -1,14 +1,33 @@
 using Compat
 
-if haskey(ENV, "CONDA_JL_HOME")
-    ROOTENV = ENV["CONDA_JL_HOME"]
-elseif isfile("deps.jl")
-    include("deps.jl")
-else
-    ROOTENV = abspath(dirname(@__FILE__), "usr")
+module DefaultDeps
+    if isfile("deps.jl")
+        include("deps.jl")
```

Comment:
If this comes from a previous version, `MINICONDA_VERSION` will not be set and will throw an error. See https://github.com/JuliaPy/Conda.jl/issues/85